test/utils/format-date-test.js

```diff
@@ -23,4 +24,60 @@ describe('utils/format-date', () => {
       assert.strictEqual(actual, false, `didn't return false as expected`)
     }
   })
+
+  test({
```

Comment:
This is awesome.

Reply:
You need to run 'restore' on the fake timers to prevent side affects (or effects?, whatever you get it). 

See: https://github.com/AugurProject/augur/blob/seadragon/src/modules/market/components/common/time-remaining-indicator/time-remaining-indicator.test.jsx#L34

Reply:
thought about that but I recreate the fake time for each unit test, your saying that just using `sinon.useFakeTimers(...)` can affect the other unit tests and I need to set to global variable and call `restore`?

Reply:
thought about that but I recreate the timer for each unit test.

Reply:
There is a magic way to get sinon to track and cleanup on it's own but I can never seem to find the documentation.